UI/WCTDataTreeTabSample/WCTDataTreeTabSample/WCTDataTreeTabSample.Shared/App.xaml.cs

```diff
@@ -1,234 +1,230 @@
-		{
+        }
+
+        /// <summary>
+        /// Invoked when the application is launched normally by the end user.  Other entry points
+        /// will be used such as when the application is launched to open a specific file.
+        /// </summary>
+        /// <param name="args">Details about the launch request and process.</param>
+        protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
+        {
 #if DEBUG
-			if (System.Diagnostics.Debugger.IsAttached)
-			{
-				// this.DebugSettings.EnableFrameRateCounter = true;
-			}
+            if (System.Diagnostics.Debugger.IsAttached)
+            {
+                // this.DebugSettings.EnableFrameRateCounter = true;
+            }
 #endif
 
-			InitializeMaterialStyles();
-
-#if NET5_0 && WINDOWS
-			var window = new Window();
-			window.Activate();
+#if NET6_0_OR_GREATER && WINDOWS && !HAS_UNO
+            _window = new Window();
+            _window.Activate();
 #else
-			var window = Windows.UI.Xaml.Window.Current;
+            _window = Microsoft.UI.Xaml.Window.Current;
 #endif
 
-			var rootFrame = window.Content as Frame;
+            var rootFrame = _window.Content as Frame;
 
-			// Do not repeat app initialization when the Window already has content,
-			// just ensure that the window is active
-			if (rootFrame == null)
-			{
-				// Create a Frame to act as the navigation context and navigate to the first page
-				rootFrame = new Frame();
+            // Do not repeat app initialization when the Window already has content,
+            // just ensure that the window is active
+            if (rootFrame == null)
+            {
+                // Create a Frame to act as the navigation context and navigate to the first page
+                rootFrame = new Frame();
 
-				rootFrame.NavigationFailed += OnNavigationFailed;
+                rootFrame.NavigationFailed += OnNavigationFailed;
 
-				if (e.PreviousExecutionState == ApplicationExecutionState.Terminated)
-				{
-					//TODO: Load state from previously suspended application
-				}
+                if (args.UWPLaunchActivatedEventArgs.PreviousExecutionState == ApplicationExecutionState.Terminated)
+                {
+                    // TODO: Load state from previously suspended application
+                }
 
-				// Place the frame in the current Window
-				window.Content = rootFrame;
-			}
+                // Place the frame in the current Window
+                _window.Content = rootFrame;
+            }
 
-#if !(NET5_0 && WINDOWS)
-			if (e.PrelaunchActivated == false)
+#if !(NET6_0_OR_GREATER && WINDOWS)
+            if (args.UWPLaunchActivatedEventArgs.PrelaunchActivated == false)
 #endif
-			{
-				if (rootFrame.Content == null)
-				{
-					// When the navigation stack isn't restored navigate to the first page,
-					// configuring the new page by passing required information as a navigation
-					// parameter
-					rootFrame.Navigate(typeof(Shell), e.Arguments);
```

Comment:
@aesteves900 this sample is broken because this has been changed. The sample should navigate to Shell no to MainPage. Please remove MainPage and revert to navigating to Shell